anomalib/data/mvtec.py

```diff
@@ -384,6 +384,8 @@ def prepare_data(self) -> None:
                     filename=self.root / dataset_name,
                     reporthook=progress_bar.update_to,
                 )
+            logger.info("Checking hash")
+            hash_check(self.root / dataset_name, "eefca59f2cede9c3fc5b6befbfec275e")
```

Comment:
quite minor, but btech has `zip_filename`, would be good if the two has consistent implementation, (ie., both has `zip_filename` or path)